flatm8-ui/src/app/container/register/register.component.html

```diff
@@ -8,8 +8,13 @@
   <h4>Please fill out the form. The fields marked with * must be filled.</h4>
   <form role="form" (submit)="register()" ngNativeValidate>
     <div class="form-group">
-      <label for="username">Email*:</label> <input type="email"
+      <label for="username">Username*:</label> <input type="userName"
```

Comment:
Input type here means the type and userName is not a valid type.
Check here: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input
Since this is a basic String I think it should be "text"